### PR TITLE
Publish beta build with new update URLs

### PR DESCRIPTION
**Pull request by atropnikov**
#622 
Auxiliary commit before publishing beta.

### CODE REVIEW

File: Extension/browser/firefox/install.rdf [hunk 1]
```diff
@@ -27,7 +27,7 @@
             <Description>
                 <em:id>{ec8030f7-c20a-464f-9b0e-13a3a9e97384}</em:id>
                 <em:minVersion>30.0</em:minVersion>
-                <em:maxVersion>45.*</em:maxVersion>
+                <em:maxVersion>*</em:maxVersion>
```
Comment (Mizzick) [inline Extension/browser/firefox/install.rdf]: are you sure it will work without defined number?

Comment (atropnikov) [inline Extension/browser/firefox/install.rdf]: Yes, it works. Although, you're right. It had better specify version like `58.*`

Comment (atropnikov) [inline Extension/browser/firefox/install.rdf]: Oh, it's ok. I found a proof.
https://addons.mozilla.org/en-US/firefox/pages/appversions/

**Comment by atropnikov** [comment]
Sorry guys, it's unnecessary commit. 
We can directly set URLs to the new version in `chrome.adtidy.org`

**Pull request closed without merging**